crates/rust-analyzer/src/config.rs

```diff
@@ -145,6 +145,8 @@ config_data! {
         inlayHints_parameterHints: bool     = "true",
         /// Whether to show inlay type hints for variables.
         inlayHints_typeHints: bool          = "true",
+        /// Whether inlay hints font size should be smaller than editor's font size.
+        inlayHints_smallerHints: bool       = "true",
```

Comment:
@matklad do you have any opinion about whether to include client- (or Code-) only configuration in `config.rs`?

Reply:
Good catch, fixed in https://github.com/rust-analyzer/rust-analyzer/pull/8683/commits/855f07202896508d288dba77a89df4364670adf5